packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -23,6 +23,22 @@
 	overflow: hidden;
 }
 
+.edit-site-typography-panel__full-width-control {
+	grid-column: 1 / -1;
+	max-width: 100%;
+}
```

Comment:
Perhaps this style is not being applied anywhere.

Reply:
Thanks for pointing that out. This was added accidentally when I merged the latest trunk branch changes. Removed here: [`73617a8` (#62328)](https://github.com/WordPress/gutenberg/pull/62328/commits/73617a8c5ae2822195628f09794ff43e4b7564c7)